numberformat/diff.emu

```diff
@@ -1004,7 +1004,7 @@
     <emu-clause id="sec-tointlmathematicalvalue" aoid="ToIntlMathematicalValue">
       <h1>ToIntlMathematicalValue ( _value_ )</h1>
       <p>
-        The abstract operation ToIntlMathematicalValue takes argument _value_. It returns _value_ converted to an <dfn id="intl-mathematical-value">Intl mathematical value</dfn>, which is a mathematical value together with *+&infin;*, *-&infin;*, *NaN*, and *-0*. This abstract operation is similar to <emu-xref href="#sec-tonumeric"></emu-xref>, but a mathematical value can be returned instead of a Number or BigInt, so that exact decimal values can be represented. The following steps are taken:
+        The abstract operation ToIntlMathematicalValue takes argument _value_. It returns _value_ converted to an <dfn id="intl-mathematical-value">Intl mathematical value</dfn>, which is a mathematical value together with *+&infin;*, *-&infin;*, *NaN*, and *-0*<sub>𝔽</sub>. This abstract operation is similar to <emu-xref href="#sec-tonumeric"></emu-xref>, but a mathematical value can be returned instead of a Number or BigInt, so that exact decimal values can be represented. The following steps are taken:
```

Comment:
I actually think this is fine, just needs to be worded more clearly: the point of this paragraph is to introduce a _new_ type, which is the set of mathematical values together with four additional distinguished points. It's fine to label one of those points `*-0*`; it's just an arbitrary label. Labeling it `*-0*<sub>𝔽</sub>` is probably more confusing.

But it does need to be clear that this is _introducing_ a new point to be called `*-0*`, which I don't think it is.

(That said, if you want to do things with the value other than say "is it -0", it will need to be given more explicit semantics. At a quick glance, I think `PartitionNumberPattern` needs to have an explicit case for -0, but it seems like it's not otherwise being used for arithmetic.)

Reply:
I hadn't considered that, but I think `*-0*` is a very misleading label for this domain. I would much prefer a more distinctive label so that the intention would be clear wherever it was used outside of this abstract operation. For example, `~negative-zero~`.

That said, regardless of what we choose for this particular point, "Intl Mathematical Value" will still contain both mathematical values and Number values. If we use an arbitrary label rather than `*-0*<sub>𝔽</sub>` (another Number value), that will tend to complicate the type.

Reply:
Yes, an Intl mathematical value is a mathematical value with four additional discrete values.  I am happy to adopt a different notational style for those values.  Using a spellout with `~~` is fine with me.  We should do it to all four, though, not just negative zero.

Perhaps:

- `*-0*` becomes `~negative-zero~`
- `*NaN*` becomes `~not-a-number~`
- `*+&infin;*` becomes `~positive-infinity~`
- `*-&infin;*` becomes `~negative-infinity~`